CMakeLists.txt

```diff
@@ -102,6 +102,8 @@ option(BUILD_EXAMPLES
     "Build example programs form the blosc compression library" ON)
 option(BUILD_PLUGINS
     "Build plugins programs form the blosc compression library" ON)
+option(BUILD_LITE
+    "Build a lite version of the blosc library." OFF)
```

Comment:
I suggest to be more explicit and say "Build a lite version (only with BloscLZ and LZ4/LZ4HC) of the blosc library."